client/asset/dcr/rpcwallet.go

```diff
@@ -486,7 +486,8 @@ func (w *rpcWallet) ExternalAddress(ctx context.Context, acctName string) (stdad
 
 // SignRawTransaction signs the provided transaction using rpc RawRequest.
 // Part of the Wallet interface.
-func (w *rpcWallet) SignRawTransaction(ctx context.Context, baseTx *wire.MsgTx) (*wire.MsgTx, error) {
+func (w *rpcWallet) SignRawTransaction(ctx context.Context, inTx *wire.MsgTx) (*wire.MsgTx, error) {
+	baseTx := inTx.Copy()
```

Comment:
OK if this is just defensive, but `baseTx` is never modified by this method, only serialized, so the copy seems to be superfluous at present.